cdap-ui/app/cdap/components/DataPrep/store/DataPrepActionCreator.js

```diff
@@ -120,3 +123,52 @@ export function setWorkspace(workspaceId) {
       });
   });
 }
+
+export function getWorkspaceList(workspaceId) {
+  let namespace = NamespaceStore.getState().selectedNamespace;
+
+  return Rx.Observable.create((observer) => {
```

Comment:
Do we need to return an observable here? I am seeing `getWorkspaceList().subscribe()` empty subscribes in this PR. Are there any subscribers that listens to observable returned by getWorkspaceList? I see `observer.onNext(workspaceList);` but not sure if there are subscribers.